src/cmd/run.js

```diff
@@ -44,16 +47,18 @@ export type WatcherCreatorParams = {|
   desktopNotifications?: typeof defaultDesktopNotifications,
   ignoreFiles?: Array<string>,
   createFileFilter?: FileFilterCreatorFn,
+  addonReload?: typeof defaultAddonReload,
 |};
 
+
 export type WatcherCreatorFn = (params: WatcherCreatorParams) => Watchpack;
 
 export function defaultWatcherCreator(
   {
     addonId, client, sourceDir, artifactsDir, ignoreFiles,
```

Comment:
it looks like `addonId` and `client` are also unused in defaultWatcherCreator now (they are parameters of the `addonReload`)